src/types/settings.ts

```diff
@@ -3,7 +3,7 @@ import {
   ProxySettingsSchema,
   RecorderSettingsSchema,
   UsageReportSettingsSchema,
-} from '@/schemas/appSettings'
+} from '@/schemas/settings/v2'
```

Comment:
To avoid updating imports every time we introduce a new version we should add export to `schemas/settings/index.ts`
```ts
export * from './v2'
```